src/blob/mod.rs

```diff
@@ -136,7 +136,10 @@
 //!
 //! // Insert another BLOB, this time using a parameter passed in from
 //! // rust (potentially with a dynamic size).
-//! db.execute("INSERT INTO test_table (content) VALUES (?)", &[ZeroBlob(64)])?;
+//! db.execute(
```

Comment:
Sorry about these, I forgot you use some unstable rustfmt settings. I'll think about a way to avoid this in the future, even if it's just running it on nightly. I mostly don't want to impose that barrier for contributors, but dont mind it that much myself.